apps/web/src/components/forms/notification-form.tsx

```diff
@@ -159,28 +189,68 @@ export function NotificationForm({
                 </FormItem>
               )}
             />
-            <FormField
-              control={form.control}
-              name="data"
-              render={({ field }) => (
-                <FormItem className="sm:col-span-full">
-                  <FormLabel>Email</FormLabel>
-                  <FormControl>
-                    <Input
-                      type="email"
-                      placeholder="dev@documenso.com"
-                      {...field}
-                    />
-                  </FormControl>
-                  <FormDescription>The data required.</FormDescription>
-                  <FormMessage />
-                </FormItem>
-              )}
-            />
+            {watchProvider && (
+              <FormField
+                control={form.control}
+                name="data"
+                render={({ field }) => (
+                  <FormItem className="sm:col-span-full">
+                    {/* make the first letter capital */}
+                    <div className="flex items-center justify-between">
+                      <FormLabel>
+                        {watchProvider[0].toUpperCase() +
+                          watchProvider.slice(1)}
+                      </FormLabel>
+                      {watchProvider !== "email" && (
+                        <Link
+                          href={`https://docs.openstatus.dev/integrations/${watchProvider}`}
+                          target="_blank"
+                          className="text-muted-foreground text-xs"
+                        >
+                          How to setup your {watchProvider} webhook
+                        </Link>
+                      )}
```

Comment:
Lets move that into the `FormDescription` and use the default styling of it. Maybe add an `underline`. Btw, you don't need `next/link` here as it opens a new tab.